### ISSUE TITLE
Broken Links found for install-configure

### ISSUE THREAD

**Issue by tr0njavolta** [crossplane/docs#813]
### What's Wrong?

A 404 from https://docs.crossplane.io/latest/getting-started/install-configure.html
Should be https://docs.crossplane.io/latest/software/  instead.

This URL structure is found in [content/media/snippets/package/gcp/crossplane.yaml](https://github.com/crossplane/docs/blob/8ebeec192e7e2389e875dca49754c2f49587a784/content/media/snippets/package/gcp/crossplane.yaml#L16), [content/media/snippets/package/aws/crossplane.yaml](https://github.com/crossplane/docs/blob/8ebeec192e7e2389e875dca49754c2f49587a784/content/media/snippets/package/aws/crossplane.yaml#L16), [content/media/snippets/package/azure/crossplane.yaml](https://github.com/crossplane/docs/blob/8ebeec192e7e2389e875dca49754c2f49587a784/content/media/snippets/package/azure/crossplane.yaml#L16), [content/media/snippets/package/aws-with-vpc/crossplane.yaml](https://github.com/crossplane/docs/blob/8ebeec192e7e2389e875dca49754c2f49587a784/content/media/snippets/package/aws-with-vpc/crossplane.yaml#L17)

I'm not sure what the snippets directory is used for, but the entire crossplane docs domain is getting crawled by the new chatbot on https://www.upbound.io

**Comment by plumbis** [comment]
Snippets is a hold over of the legacy docs site. @ytsarev had some use cases for it, but it's not used inside the docs any longer.

**Comment by ytsarev** [comment]
Yep, in the old doc setup, I used snippets as a source of examples. Now, all required code samples can be found on the documentation website, so I no longer need them.
